src/app/features/course-info/course-info.module.ts

```diff
@@ -1,11 +1,14 @@
 import { NgModule } from '@angular/core';
 import { CourseInfoComponent } from './course-info.component';
 import { SharedModule } from '@app/shared/shared.module';
+import { RouterModule, Routes } from '@angular/router';
+
+const courseInfoRoute: Routes = [{ path: '', component: CourseInfoComponent }];
 
 @NgModule({
   declarations: [CourseInfoComponent],
-  imports: [SharedModule],
+  imports: [SharedModule, RouterModule.forChild(courseInfoRoute)],
```

Comment:
I guess it would be more appropriate to create the routing for the module (just like it is done with AppModule)